src/protocol.cpp

```diff
@@ -2879,7 +2879,9 @@ bool CProtocol::ParseMessageFrame ( const CVector<uint8_t>& vecbyData,
 
     // Extract actual data -----------------------------------------------------
 
+// clang-format off
 // TODO this memory allocation is done in the real time thread but should be
+// clang-format on
```

Comment:
Could be moved that to the end of the comment block for better readability?

Reply:
ah, missed that one... now it's fixed